docs/examples/ex_match.rst

```diff
@@ -138,8 +140,8 @@ drifts.  We aren't directly comparing the values, since the test file is filled
 with randomly generated values that aren't realistic.
 
 ::
-   
-   import matplotlib as mpl
```

Comment:
since "mpl" is referenced in the plotting example, I think this should stay in.

Reply:
We need to make it consistent, as `plt` is elsewhere

Reply:
They both need to be in, and now they are.